src/fs.ts

```diff
@@ -65,4 +67,25 @@ export default class FsAsync {
 			});
 		});
 	}
+
+	static tmpFile(prefix: string, ext: string) {
```

Comment:
I think we should name this for what it actually does: "createTmpFile", "makeTmpFile" or similar.